src/lib/avax.js

```diff
@@ -26,7 +26,7 @@ class AvaxLib {
         transactions(
           address: "${addr}"
           offset: 0
-          orderBy: { acceptedAt: "desc" }
+          orderBy: { acceptedAt: "asc" }
```

Comment:
Apparently the orderBy is not working, it's always ordered from the oldest to the newest. To solve it the array it's reversed afterwards.